go/teams/keys.go

```diff
@@ -363,3 +363,33 @@ func derivedSecret(secret keybase1.PerTeamKeySeed, context string) []byte {
 	digest.Write([]byte(context))
 	return digest.Sum(nil)[:32]
 }
+
+// Decrypt a single prev secretbox.
+// Takes a prev to decrypt and the seed of the successor generation.
+// For example (prev[3], seed[4]) -> seed[4]
+func decryptPrevSingle(ctx context.Context,
+	prevToDecrypt prevKeySealedEncoded, successor keybase1.PerTeamKeySeed) (*keybase1.PerTeamKeySeed, error) {
```

Comment:
see if you can merge this with the decrypt prev function that's already in there. looks very familiar